uint8 TYPE_REALSENSE=0
uint8 TYPE_KINECT2=1

uint64 session_id
string id
uint8  type
string serial

---

uint8 STATUS_OK=0
uint8 STATUS_ERROR=1

uint8  status
string message
